### PR TITLE
5.13.1 Release and Promotion

### PR DESCRIPTION
**Pull request by Blue0ctober**
## Description

5.13.1 release!

### CODE REVIEW

File: content/sensu-go/5.13/release-notes.md [hunk 1]
```diff
@@ -38,6 +39,16 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
 
+## 5.13.1 release notes
+
+**September 10, 2019** &mdash; The latest release of Sensu Go, version 5.13.1, is now available for download.
+This release is a stability release with bug fixes for multi-build asset definitions causing a panic when no matching filters are found.
+See the [upgrade guide][1] to upgrade Sensu to version 5.11.1.
+
+**FIXES:**
+
+- Multi-build asset definitions with no matching filters will no longer cause a panic.
```
Comment (ccressent) [inline content/sensu-go/5.13/release-notes.md:50]: There is also a fix for OIDC not working on 5.13.0, see:
https://github.com/sensu/sensu-enterprise-go/commit/0dd79f311f85ab9a2b337b4ccea9ed4671357a7b#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR8-R11

Comment (rgeniesse) [inline content/sensu-go/5.13/release-notes.md:50]: Isn't there a OIDC fix as well?

File: content/sensu-go/5.13/installation/install-sensu.md [hunk 2]
```diff
@@ -254,13 +254,13 @@ sudo yum install sensu-go-agent
 
 {{< highlight "Windows" >}}
 # Download the Sensu agent for Windows amd64
-Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.13.0/sensu-go-agent_5.13.0.5887_en-US.x64.msi  -OutFile "$env:userprofile\sensu-go-agent_5.13.0.5887_en-US.x64.msi"
+Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.13.1/sensu-go-agent_5.13.1.5887_en-US.x64.msi  -OutFile "$env:userprofile\sensu-go-agent_5.13.1.5887_en-US.x64.msi"
```
Comment (ccressent) [inline content/sensu-go/5.13/installation/install-sensu.md]: Unfortunately we include a build number in the filename, which is also going to change. That is, it won't be `5887` in the filename anymore, but another number I'll get later in the process.

I see no reason to include a build number in the file name; we already reference a release's build number in our release checklist. I'll create a retro card to get rid of that number in the filename.

**Review by ccressent — CHANGES_REQUESTED** [review]

File: content/sensu-go/5.13/installation/install-sensu.md (hunk 2 above)
Comment (rgeniesse) [reply content/sensu-go/5.13/installation/install-sensu.md]: thanks @ccressent, this will almost make updating the downloads page easier as well.

Comment (ccressent) [reply content/sensu-go/5.13/installation/install-sensu.md]: Totally; that will mean you only need to know the release version number to update all those links, which can then be done in advance if one so wishes. No need to wait for eng to tell you the build number.

For this patch release though, the build number is `5957`.

Comment (Blue0ctober) [reply content/sensu-go/5.13/installation/install-sensu.md]: @ccressent to confirm it'll be `https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.13.1/sensu-go-agent_5.13.1_en-US.x64.msi`   then?

Comment (ccressent) [reply content/sensu-go/5.13/installation/install-sensu.md]: The new build number, `5957`, will still be in the file name for the files ending in `.msi`, `.rpm` and `.deb`. For example: `https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.13.1/sensu-go-agent_5.13.1.5957_en-US.x64.msi`

See the files on S3: https://s3.console.aws.amazon.com/s3/buckets/sensu.io/sensu-go/5.13.0/?region=us-west-2&tab=overview

**Review by ccressent — APPROVED** [review]

**Pull request merged**